core/amber/src/main/python/core/models/operator.py

```diff
@@ -47,6 +48,34 @@ def close(self) -> None:
         """
         pass
 
+    def process_state(self, state: State, port: int) -> Optional[State]:
+        """
+        Process an input State from the given link.
```

Comment:
As you are giving a default implementation here, please explain the default implementation in the doc.

Reply:
Fixed.